archivebox/archive_methods.py

```diff
@@ -339,11 +342,18 @@ def fetch_screenshot(link_dir, link, timeout=TIMEOUT):
     """take screenshot of site using chrome --headless"""
 
     output = 'screenshot.png'
-    cmd = [
-        *chrome_args(TIMEOUT=timeout),
-        '--screenshot',
-        link['url'],
-    ]
+    if CHROME_AVAILABLE:
+        cmd = [
+            *chrome_args(TIMEOUT=timeout),
+            '--screenshot',
+            link['url'],
+        ]
+    elif FIREFOX_AVAILABLE:
+        cmd = [
+            *firefox_args(),
+            '--screenshot',
+            link['url'],
+        ]
     status = 'succeeded'
```

Comment:
no default?

Reply:
IIRC fetch_screenshot should only be called if either CHROME_AVAILABLE or FIREFOX_AVAILABLE is true, but in the way I wrote, it would probably raise NameError on failure...

Using an `else:` instead of an `elif:` would be ok or is there a dummy cmd value? (or maybe an early error return)

Reply:
@homer-simpson